config/default.yml

```diff
@@ -1,3 +1,9 @@
+Layout/AccessModifierIndentation:
+  EnforcedStyle: outdent
+
+Layout/EmptyLinesAroundAccessModifier:
+  Enabled: false
```

Comment:
Unfortunately, this will allow:
``` ruby
class Sample
private

  def foo
    :foo
  end
end
```

Which does not match our usual coding style. Either I didn't found the correct
config, either we'll need a custom cop for that.

Reply:
Or we keep it enabled? I wouldn't be offended, by something like this, tbh:

```ruby
class Sample
  def my_public_method
  end

private

  def my_private_method
  end
end
```

Reply:
I actually prefer this style too in fact.

But I think that @thibaultdalban doesn't like it.